Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Using the official python 2.7 as a Base Image
+FROM python:2
+
+ADD . /edx-e2e-tests
+WORKDIR /edx-e2e-tests
+
+# Configuration
+RUN apt-get update
+RUN pip install paver
+
+RUN pip install virtualenv==1.10.1
+RUN pip install virtualenvwrapper
+
+RUN apt-get install git
+
+RUN pip install -r requirements/base.txt
+RUN paver install_pages
+
+# Set up environment variables
+CMD sleep 10000
```

Comment:
I understand using this for debugging, but we will need another solution for this. We need a CMD/ENTRYPOINT that will run indefinitely. The trouble is, all of the processes we want to run from within this container are short lived. At the least, you could use 'sleep infinity'. Perhaps there is a way to do this via the docker-compose file?